packages/client/hmi-client/src/assets/css/reset.css

```diff
@@ -2,6 +2,7 @@
 html,
 body {
 	overflow-x: hidden;
+	overscroll-behavior-x: none; /* Stop browser back navigation on overscroll */
```

Comment:
Wondering if this is a bit extreme - I for one tend to use the two finger scroll to go forward and back quite a bit cause it is convenient and I don't use a mouse.

Reply:
me too. Does it prevent me from going back with two finger swipe gesture?

Reply:
I agree, I use that feature a lot.